### DISCUSSION TITLE
Differences among model filterbanks? When and why to use which filterbank?

### DISCUSSION THREAD

**Discussion by denmonz** [asteroid-team/asteroid#545] [Q&A]
For all models, there's a parameter for filterbanks with the following values:
`free`, `analytic_free`, `param_sinc`, and `stft`

However, it's unclear to me exactly which filterbank would be best for my custom dataset.

Is there any literature that does ablation studies on these four different filterbank types?

I've read [Filterbank design for end-to-end speech separation by Pariente et al., 2020](https://arxiv.org/abs/1910.10400) but it's still unclear to me when to use which filterbank.

**Comment by mpariente** [comment]
I'd use free filterbank by default. And then once it works try analytic.

**Reply by denmonz** [reply]
Thank you! I'll evaluate my model and update this thread accordingly
